docs/coordinates/index.rst

```diff
@@ -427,9 +427,10 @@ The resulting timezone name could then be used with any packages that support
 time zone definitions, such as the (Python 3.9 default package) `zoneinfo
 <https://docs.python.org/3/library/zoneinfo.html>`_:
 
-.. doctest-skip::
+.. doctest-remote-data::
 
-    >>> from zoneinfo import ZoneInfo  # requires Python 3.9 or greater
+    >>> import datetime
+    >>> from zoneinfo import ZoneInfo
     >>> tz = ZoneInfo(tz_name)
```

Comment:
If `tz_name` was defined in a setup block then this wouldn't have to be a remote test.

Reply:
Done!

Reply:
@eerovaher, errors on the cron jobs only.

Reply:
We can fix this in a followup.

Reply:
I'd prefer this example to be included in the non-remote tests, but with the feature freeze approaching it is indeed best to leave that for a followup. Having it included in the remote tests is an improvement already.

Reply:
The `of_address` always needs remote data access, so I don't understand your original comment above, but this is unrelated to dropping Python 3.8, so please open a follow-up issue now so we don't forget about it. Thanks!

Reply:
There's two related examples here close to each other. The first example shows how to get the name of a timezone and that can be left as a remote test. The second example demonstrates using a `ZoneInfo` object created from the timezone name and can be made a non-remote test if `tz_name` is defined in a setup block.

Reply:
I opened follow-up issue at https://github.com/astropy/astropy/issues/14646